package.json

```diff
@@ -25,19 +25,25 @@
   ],
   "author": "Joel Denning",
   "license": "MIT",
-  "dependencies": {},
+  "dependencies": {
+    "custom-event": "^1.0.1"
```

Comment:
Now that we're using webpack to build single-spa, we need custom event as an npm dependency